terraform/modules/aws-brig-email-sending/resources.dns.tf

```diff
@@ -0,0 +1,16 @@
+resource "aws_route53_record" "domain_amazonses_verification_record" {
```

Comment:
This file could use some comments to explain what it does. Alternatively, the REAMDE should explain the DKIM setup (or link to documentation on this elsewhere)